views/logs.html

```diff
@@ -0,0 +1,22 @@
+<div class="table-wrapper">
```

Comment:
Okay, organize however you want. I was just confused on the need for the
src folder.
On Mar 8, 2015 8:29 PM, "Alex Memering" <notifications@github.com> wrote:

> In views/logs.html
> <https://github.com/Memeriaj/CarLog/pull/1#discussion_r26011263>:
>
> > @@ -0,0 +1,22 @@
> > +<div class="table-wrapper">
>
> See I'm not really sure I'm a fan of organizing it this way. I guess since
> the files are named the same then its easy to find its counterpart, but I
> still think it makes more sense for each to have its own folder, especially
> if one portion was complicated enough to warrant multiple JavaScript files.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Memeriaj/CarLog/pull/1/files#r26011263>.
>